nncf/openvino/statistics/collectors.py

```diff
@@ -155,115 +155,91 @@ def get_output_names(self, target_node_name: str, port_id: int) -> List[str]:
 
 
 class OVMinReducer(MinReducer):
```

Comment:
Should `NAME` attribute be removed in the `class OVNoopReducer(NoopReducer): `?

Reply:
Good catch!